### ISSUE TITLE
Debugger bubbles not animated in Godot 4.1

### ISSUE THREAD

**Issue by bitbrain** [bitbrain/beehave#233]
Godot version: 4.1
Beehave Version: 2.7.6

**Describe the bug**
When running the debugger in Godot 4.1, it works and appears correctly but the bubbles do not seem to show/animate.

**Comment by peterpants2** [comment]
I'm not sure if it's related but my debug tree hasn't worked properly since 4.0 but it's probably something I've done to the files or configuration. I've also written some custom behavior leafs that might be causing issues. I'm not sure.

![image](https://github.com/bitbrain/beehave/assets/32931669/84dc7862-f2fa-443f-b6db-1b9e20845355)

  ./core/variant/variant_utility.cpp:905 - Can't find Beehave Plugin
  res://addons/beehave/debug/graph_node.gd:53 - Invalid get index 'empty' (on base: 'Nil').
  
  This is the initial error that's spammed maybe 10-15 times and then the other error spams.. many more times :)

**Comment by LRP500** [comment]
Same issue here and I may have a bit more info on how to reproduce and fix:

I am using the **Debug Draw 3D** plugin installed from the AssetLib.
Without this plugin, this method in `utils.gd` works fine and the debugger works flawlessly.
```
static func get_plugin() -> EditorPlugin:
	var tree: SceneTree = Engine.get_main_loop()
	return tree.get_root().get_child(0).get_node_or_null("BeehavePlugin")
```
But with **Debug Draw 3D** installed it seems that `get_child(0)` doesn't point to the correct node anymore so **BeehavePlugin** cannot be found.
Replacing `get_child(0)` with `get_child(1)` in my case seems to work but it's probably duct-tape.

**Comment by bitbrain** [comment]
Fixed by #345
